backend/src/resolvers/GroupResolver.ts

```diff
@@ -56,6 +57,19 @@ export default class GroupResolver {
     Object.assign(newGroup, data);
     newGroup.owner = ctx.currentUser;
 
+    // Validate emails and fetch users
+    if (data.members && data.members.length > 0) {
+      const members = [];
+      for (const email of data.members) {
+        const user = await findUserByEmail(email);
+        if (!user) {
+          throw new GraphQLError(`User with email ${email} not found`);
+        }
+        members.push(user);
+      }
+      newGroup.members = members; // Add the members to the group
+    }
```

Comment:
```suggestion
    if (data.members && data.members.length > 0) {
      const members = [];
      const invalidEmails = [];
    
      for (const email of data.members) {
        const user = await findUserByEmail(email);
        if (!user) {
          invalidEmails.push(email);
        } else {
          members.push(user);
        }
      }
    
      if (invalidEmails.length > 0) {
        throw new GraphQLError(`USER_WITH_EMAIL_NOT_FOUND: ${invalidEmails.join(', ')}`);
      }
    
      newGroup.members = members;
    }
```

ça va te permettre de gérer si plusieurs adresses mail sont aussi invalides